src/pages/Host.tsx

```diff
@@ -77,7 +78,7 @@ export const Host: React.FC = () => {
 
   usePageTitle(`Host${hostId ? ` - ${hostId}` : ""}`);
 
-  const canRestartJasper =
+  const canRestartJasperOrReprovision =
     host?.status === "running" &&
     (bootstrapMethod === "ssh" || bootstrapMethod === "user-data");
```

Comment:
I assumed the condition for reprovisioning from [here](https://github.com/evergreen-ci/evergreen/blob/dd69abc6b54990e27968dc5488aa2c839adc6647/public/static/js/host_admin.js#L15-L17)